cwms-data-api/src/main/java/cwms/cda/data/dto/project2/project/ProjectLock.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2024 Hydrologic Engineering Center
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package cwms.cda.data.dto.project2.project;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.data.dto.CwmsDTO;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV1;
+
+@JsonDeserialize(builder = ProjectLock.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+@FormattableWith(contentType = Formats.JSON, formatter = JsonV1.class)
+public class ProjectLock extends CwmsDTO {
+    // officeId held by CwmsDTO
+    private final String projectId;
+    private final String applicationId;
+    private final String acquireTime;
```

Comment:
Don't you want this to be an Instant?

Reply:
Instant sounds good to me. Here is why I didn't.  The pl/sql has a timestamp and goes thru some effort to format the timestamp into the requested timezone.   I don't totally 100% understand what REGI does with project lock or why the pl/sql was letting people format the acquireTime in different timezones.  It seemed weird to me to let users request certain tz and then have CDA parse that string back into a timestamp (Instant) just to serialize it into a String to send over the network.  I could maintain the timezone by parsing into ZonedDateTime.   It was at this point I started to doubt myself and decided that if the pl/sql was returning varchar I should just hold/return String and get out of the way.

Reply:
But maybe the right answer is to do Instant and not expose the timezone_id request parameter and have CDA only deal with Instant and utc.

Reply:
So this is an ISO string then? That's fine too.

Reply:
Current usage is UTC only, this will be update accordingly as an Instant.

Reply:
Current usage:  https://bitbucket.hecdev.net/projects/CWMS/repos/hec-cwms-data-access/browse/hec-db-jdbc/src/main/java/wcds/dbi/oracle/cwms/CwmsProjectJdbcDao.java#300